api/src/pcapi/routes/serialization/collective_bookings_serialize.py

```diff
@@ -329,6 +329,9 @@ def serialize_collective_booking(collective_booking: models.CollectiveBooking) -
     "Prix de la réservation",
     "Statut de la réservation",
     "Date et heure de remboursement",
+    "uai de l'institution",
+    "nom de l'institution",
+    "type de l'institution",
```

Comment:
```suggestion
    "uai",
    "nom de l'établissement",
```

Le type d'établissement doit être fusionné dans le nom de l'établissement scolaire.